.github/workflows/cs-fixer.yml

```diff
@@ -0,0 +1,14 @@
+name: CS-Fixer
```

Comment:
I would merge the cs-fixer and php-stan into a single `php-lint.yml` workflow with multiple jobs 🤔

Reply:
I would keep it as it is for the moment to be consistent with the other bundle https://github.com/shopware/service-bundle-symfony